CONTRIBUTING.rst

```diff
@@ -64,7 +64,7 @@ Ready to contribute? Here's how to set up ``conda-devenv`` for local development
 
 3. Create a new conda environment for developing::
 
-    $ conda create -n devenv --file requirements_dev.txt
+    $ conda create -c conda-forge -n devenv --file requirements_dev.txt
```

Comment:
"conda-forge" channel should not be necessary for this.
Did you have problems with this? Like some missing package?

Reply:
Yeah, these are missing when trying to install from defaults on Linux:
```
  - tox
  - bumpversion
  - pre-commit
  - pytest-datadir[version='>=1.0']
```